bundles/org.eclipse.emfcloud.modelserver.common/src/org/eclipse/emfcloud/modelserver/common/patch/PatchUtil.java

```diff
@@ -0,0 +1,98 @@
+/********************************************************************************
+ * Copyright (c) 2022 STMicroelectronics..
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.emfcloud.modelserver.common.patch;
+
+import java.util.stream.StreamSupport;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+/**
+ * Utility methods for Json Patch objects.
+ */
+public final class PatchUtil {
+
+   // Json Patch Operation Constants
+
+   public static final String OP = "op";
+   public static final String TEST = "test";
+   public static final String MOVE = "move";
+   public static final String REMOVE = "remove";
+   public static final String ADD = "add";
+   public static final String REPLACE = "replace";
+   public static final String ANY_INDEX = "-";
+   public static final String PATH = "path";
+   public static final String VALUE = "value";
+
+   // Model Patch Constants
+
+   /**
+    * The "modelUri" attribute, for ModelPatch elements.
+    */
+   public static final String MODEL_URI = "modelUri";
+
+   /**
+    * The "patch" attribute, for ModelPatch elements.
+    */
+   public static final String PATCH = "patch";
+
+   /**
+    * Test if the specified node is a Json Patch (an array of Operations).
+    *
+    * @param jsonNode
+    *                    The node to test.
+    * @return
+    *         <code>true</code> if the jsonNode is a Json Patch, <code>false</code> otherwise.
+    */
+   public static boolean isPatch(final JsonNode jsonNode) {
+      return jsonNode.isArray() && PatchUtil.isPatch((ArrayNode) jsonNode);
+   }
+
+   /**
+    * Test if the specified array is a Json Patch (an array of Operations).
+    *
+    * @param jsonNode
+    *                    The node to test.
+    * @return
+    *         <code>true</code> if the jsonNode is a Json Patch, <code>false</code> otherwise.
+    */
+   public static boolean isPatch(final ArrayNode jsonNode) {
```

Comment:
```suggestion
   public static boolean isPatch(final ArrayNode arrayNode) {
```
Maybe change the name here? And in the method and the doc. But i am also fine with leaving it as is.